apm.(*Tracer).Close panic

Running into this panic while working on integration tests:

```
panic: close of closed channel

goroutine 67 [running]:
github.com/elastic/apm-server/vendor/go.elastic.co/apm.(*Tracer).Close(0xc000154000)
	go/src/github.com/elastic/apm-server/vendor/go.elastic.co/apm/tracer.go:306 +0xc9
github.com/elastic/apm-server/beater.initTracer(0x4b0c102, 0xf, 0x4b0c102, 0xf, 0x4b06eb7, 0x5, 0x0, 0x0, 0x0, 0x0, ...)
	go/src/github.com/elastic/apm-server/beater/beater.go:176 +0x63
github.com/elastic/apm-server/beater.(*beater).Run(0xc000330870, 0xc00036c000, 0x0, 0x0)
	go/src/github.com/elastic/apm-server/beater/beater.go:125 +0x10f
```

Appears to be related to the recent elastic/apm-server#1738 change.

--- a/beater/beater.go
+++ b/beater/beater.go
@@ -38,6 +38,10 @@ import (
 	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
 )
 
+func init() {
+	apm.DefaultTracer.Close()
+}
+
 type beater struct {
 	config  *Config
 	mutex   sync.Mutex // guards server and stopped
@@ -173,7 +177,6 @@ func (bt *beater) Run(b *beat.Beat) error {
 // initTracer configures and returns an apm.Tracer for tracing
 // the APM server's own execution.
 func initTracer(info beat.Info, config *Config, logger *logp.Logger) (*apm.Tracer, net.Listener, error) {
-	apm.DefaultTracer.Close()
 	if !config.SelfInstrumentation.isEnabled() {
 		os.Setenv("ELASTIC_APM_ACTIVE", "false")
 		logger.Infof("self instrumentation is disabled")

